pkg/services/sqlstore/user.go

```diff
@@ -523,6 +526,16 @@ func DeleteUser(cmd *models.DeleteUserCommand) error {
 }
 
 func deleteUserInTransaction(sess *DBSession, cmd *models.DeleteUserCommand) error {
+
+	//Check if user exists
+	user := models.User{Id: cmd.UserId}
+	has, err := sess.Get(&user)
+	if err != nil {
+		return err
+	} else if !has {
```

Comment:
More idiomatic I think:
```suggestion
	}
	if !has {
```